mllib/src/test/scala/org/apache/spark/mllib/classification/NaiveBayesSuite.scala

```diff
@@ -85,19 +92,66 @@ class NaiveBayesSuite extends FunSuite with MLlibTestSparkContext {
     assert(numOfPredictions < input.length / 5)
   }
 
-  test("Naive Bayes") {
+  def validateModelFit(piData: Array[Double], thetaData: Array[Array[Double]], model: NaiveBayesModel) = {
```

Comment:
line too long (limit to <= 100 chars).  (This may not be checked by Scala style since it's a test suite, but it's still nice to follow.)